research/conditional/utils/argparse.py

```diff
@@ -22,6 +22,9 @@ def introduce_parser_arguments(
     parser.add_argument("--every_other_layer", action="store_true")
     parser.add_argument("--standard_ff_first", action="store_true")
     parser.add_argument("--no_ff", action="store_true")
+    parser.add_argument(
+        "--token_choice_logic", type=str, choices=["relu", "swi_glu"], default="relu"
```

Comment:
Maybe `swiglu` instead of `swi_glu`?

Reply:
I will leave it as swi_glu for now to keep it consistent with implementation in normal (non-expert) FF. Maybe seperate pr to rename both?

Reply:
Ahh I see. We can keep it that way then.